doc/source/reference/functions/time_classify_day.rst

```diff
@@ -8,11 +8,111 @@
 Summary
 -------
 
+``time_classify_day`` rounds time to a day unit.
+
+This rounded ``2020-01-30 11:50:11.000000`` and ``2020-01-30 22:50:11.000000`` to ``2020-01-30 00:00:00.000000``.
+
 Syntax
 ------
 
+``time_classify_day`` has one parameter::
```

Comment:
Could you use "This function has only one parameter" to reuse existing translation?